pkg/commands/helpers_readwrite.go

```diff
@@ -17,8 +17,7 @@ limitations under the License.
 package commands
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
```

Comment:
FYI I think we should use the functionality in [xerrors](https://godoc.org/golang.org/x/xerrors) which did make it into go 1.13, specifically `fmt.Errorf("foo: %w", err)`